crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -102,9 +102,20 @@ pub struct BloomSettings {
     /// configured in a non-energy-conserving way,
     /// otherwise set to [`BloomCompositeMode::EnergyConserving`].
     pub composite_mode: BloomCompositeMode,
+
+    /// Maximum size of each dimension for the largest mipchain texture used in downscaling/upscaling.
+    /// Only tweak if you are seeing visual artifacts.
+    pub max_mip_dimension: u32,
+
+    /// UV offset for bloom shader. Ideally close to 2.0 / max_mip_dimension.
```

Comment:
```suggestion
    /// UV offset for bloom shader. Ideally close to 2.0 / `max_mip_dimension`.
```

Looks like this should fix the CI issue.